AUTHORS

```diff
@@ -1 +1,2 @@
 Timur Shemsedinov <timur.shemsedinov@gmail.com>
+Sergey Nanovsky <sergey.nanovsky@gmail.com>
```

Comment:
That would not be related to this PR :)
I am okay with @sun1x adding himself to the list in his first PR, though I would not say it is a good thing to do generally. But adding other people should definitely be done separately.